app/views/pages/primary-senior-leaders.html.erb

```diff
@@ -1,187 +1,152 @@
+              { image_url: "media/images/icons/physical_computing_kits.svg", text: "Physical computing kits"},
+              { image_url: "media/images/icons/real_world.svg", text: "Real-world contexts"},
+              { image_url: "media/images/icons/subject_matter_experts.svg", text: "Subject matter experts support"},
+            ]
+          ) %>
+        </div>
+      </div>
+
+      <div class="govuk-grid-row govuk-!-margin-top-6">
+        <div class="govuk-grid-column-two-thirds senior-leaders">
+
+          <%= render partial: 'pages/computing_quality_framework' %>
+         
+        </div>
+        <div class="govuk-grid-column-one-third">
+          <%= render AsideComponent.new(
+            title: "Guidance for offering computing",
+            text: "Our network of #{link_to('Computing Hubs', hubs_url)} is there to support you and your school to offer exemplary computing education for all.".html_safe
+          ) %>
+        </div>
       </div>
     </div>
   </div>
 </div>
 
-<%# How we can help your teachers %>
 <div class="govuk-width-container">
-  <div class='govuk-main-wrapper govuk-main-wrapper--xl'>
+  <div class='govuk-main-wrapper'>
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <h2 class='govuk-heading-l'><%= t('.how-we-help.title') %></h2>
-      </div>
-    </div>
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full govuk-!-margin-top-7">
-        <%= render BorderedCardsComponent.new(
+      <div class="govuk-grid-column-full">
+        <%= render NonBorderedCardsComponent.new(
+          class_name: 'senior-leaders',
           cards_per_row: 3,
-          class_name: 'teacher-help-cards',
           cards: [
             {
-              class_name: 'bordered-card--primary-cert',
-              title: t('.how-we-help.cards.courses.title'),
-              text: t(
-                '.how-we-help.cards.courses.text.html',
-                all_courses_link: (link_to(
-                  t('.how-we-help.cards.courses.text.all-courses-link.title'),
-                  t('.how-we-help.cards.courses.text.all-courses-link.url'),
-                  data: {
-                    event_action: 'click',
-                    tracking_page: 'Primary SLT',
-                    tracking_label: t('.how-we-help.cards.courses.text.all-courses-link.event-label')
-                  }
-                ))
-              ),
-              link: {
-                link_title: t('.how-we-help.cards.courses.link.title'),
-                link_url: t('.how-we-help.cards.courses.link.url'),
-                tracking_page: 'Primary SLT',
-                tracking_label: t('.how-we-help.cards.courses.link.event-label')
-              }
+              title: t('.objectives.cards.confidence.title'),
+              text: t('.objectives.cards.confidence.text.html',
+                  subsidies_link: (link_to('subsidies', cms_page_url('funding#primary-subsidies')))),
```

Comment:
The URL appears to be broken, I think it should be:

`cms_page_url('funding', anchor: 'primary-subsidy')` 

Note the singular spelling, and no need for hash symbol in anchor value (some that needs to be fixed elsewhere in the project).